lib/api/v3/queries/query_representer.rb

```diff
@@ -259,11 +259,9 @@ class QueryRepresenter < ::API::Decorators::Single
                  exec_context: :decorator,
                  getter: nil,
                  setter: ->(fragment:, **) {
-                   next unless represented.new_record?
+                   next if represented.persisted?
 
-                   Hash(fragment).each do |wp_id, position|
-                     represented.ordered_work_packages.build(work_package_id: wp_id, position: position)
-                   end
+                   represented.ordered_work_packages = fragment
```

Comment:
Just to double check: This does not right away persist the value since we never pass in an already persisted query into the representer anywhere, right?

Reply:
The update case still inputs an actual query resource, which is why we test for `persisted?` a line above here